integration/log.go

```diff
@@ -119,7 +124,7 @@ func RunLogIntegration(client trillian.TrillianLogClient, params TestParameters)
 
 	// Step 3 - Use get entries to read back what was written, check leaves are correct
 	glog.Infof("Reading back leaves from log ...")
-	leafMap, err := readbackLogEntries(params.treeID, client, params)
+	leafMap, err := readbackLogEntries(params.treeID, client, params, leafCounts)
```

Comment:
Haha, no I meant if you run the test client with the option where it skips queuing the map will be empty and it gets used later. Possibly it still works, not sure. It's ok for the test runs as they never set this.